patterns/design-patterns-architectural/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger=INFO, file, console
```

Comment:
Per the "for authors - 4 - implementation code (new)" doc, we should use slf4j with logback for consistency with other modules.